applications/AdjointFluidApplication/tests/test_vms_sensitivity_2d/cylinder_test_adjoint_parameters.json

```diff
@@ -138,7 +138,8 @@
         "Parameters" : {
             "position"         : [0.020957, 0.0055272, 0.0],
             "model_part_name"  : "Parts_Fluid",
-            "output_file_name" : "test_vms_sensitivity_2d/cylinder_test_adjoint_probe1.dat",
+            "output_file_name" : "cylinder_test_adjoint_probe1.dat",
+            "output_folder"    : "test_vms_sensitivity_2d",
```

Comment:
Did you compile the AdjointFluidApp and run this test?

Reply:
In the FluidDynamicApplication/tests all seems to be good:
fluid_analysis_test.py
adjoint_fluid_test.py
test_FluidDynamicsApplication.py (ran 39 tests, skipped 3)

The AdjointFluidApplication/tests there seems to be an error unrelated to the current changes.